programs/canonical-swap/src/lib.rs

```diff
@@ -276,9 +266,10 @@ pub struct InitializeWrappedToken<'info> {
             canonical_data.mint.as_ref(),
             wrapped_token_mint.to_account_info().key.as_ref()
```

Comment:
Just triple confirming - there's no way to generate some other `wrapped_token_mint` address that would generate the same output that `seeds` checks here?